src/Meditation.UI/App.axaml

```diff
@@ -1,7 +1,7 @@
 <Application xmlns="https://github.com/avaloniaui"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              x:Class="Meditation.UI.App"
-             RequestedThemeVariant="Default">
+             RequestedThemeVariant="Light">
              <!-- "Default" ThemeVariant follows system theme variant. "Dark" or "Light" are other available options. -->
```

Comment:
> [Question] Does this not work well?

UI is currently designed only for light theme. By default it inherited settings from current system. When people are testing with dark theme on their system, colors are off. This change just applies light theme for everyone